packages/atomic/src/components/common/facets/facet-search/facet-search-matches.tsx

```diff
@@ -22,13 +24,25 @@ function matchesFound(
   });
 }
 
+function matchesFoundClickableShowMoreMatches(
+  key: 'more-matches-for' | 'no-matches-found-for',
+  query: string,
+  i18n: i18n
+) {
+  return i18n.t(key, {
+    query: `<span class="font-bold italic" part="matches-query">${escape(
+      query
+    )}</span>`,
+    interpolation: {escapeValue: false},
+  });
+}
+
```

Comment:
This is temporary while before https://coveord.atlassian.net/browse/KIT-3368 (implementation for commerce) is done